go.mod

```diff
@@ -4,11 +4,11 @@ go 1.18
 
 require (
 	github.com/hashicorp-demoapp/hashicups-client-go v0.0.0-20210721190446-1df90c457bd4
-	github.com/hashicorp/terraform-plugin-docs v0.13.0
+	github.com/hashicorp/terraform-plugin-docs v0.14.0
```

Comment:
We actually have an open issue relating to this specific version of `terraform-plugin-docs v0.14.0` that can cause some weird behavior for acceptance tests. I don't think our usage in this repo will hit that bug, but probably worth not publishing with this version.

Depending on your urgency to merge this PR, you can either move back to the previous version `terraform-plugin-docs v0.13.0` or wait for us to release `v0.14.1`, which I imagine we'll do before the end of this week.

If you want more details you can read/subscribe here: https://github.com/hashicorp/terraform-plugin-docs/issues/228

Reply:
You can update this to `v0.14.1` now, approving the PR
![image](https://user-images.githubusercontent.com/8650838/222548035-18bfa4bc-c4a4-4b13-8c39-3a088d8a3738.png)